inst/qml/doeResponseSurfaceMethodology.qml

```diff
@@ -20,321 +20,281 @@ import JASP.Widgets 							1.0
 
 Form 
 {
+	usesJaspResults:							true
```

Comment:
```suggestion
```
this hasn't been necessary for a long time... are you sure you copied the right stuff?